src/pages/Articles.jsx

```diff
@@ -7,19 +8,21 @@ const Articles = () => {
   const [articles, setArticles] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   const [err, setErr] = useState(false);
+  const { topic } = useParams();
 
   useEffect(() => {
-    getArticles()
+    setIsLoading(true);
+    setErr(false);
+    getArticles({ topic })
       .then((articles) => {
-        console.log(articles);
         setArticles(articles);
         setIsLoading(false);
         setErr(false);
       })
       .catch((err) => {
```

Comment:
Great to see some error handling already! You need to allow for a topic with no articles as the BE returns an empty array in this case. As the moment, the user just gets a blank Articles screen. (eg if I navigate to /topic/some_other_topic